src/pages/ApplicationControls.jsx

```diff
@@ -0,0 +1,143 @@
+import { useEffect, useState } from 'react';
+import { useGet, useMutate } from 'restful-react';
+
+import {
+  Box,
+  Flex,
+  Heading,
+  Input,
+  FormControl,
+  FormLabel,
+  Button,
+  Alert,
+  AlertIcon,
+  AlertTitle,
+  AlertDescription,
+  CloseButton,
+} from '@chakra-ui/react';
+
+import CountdownTimer from '../components/CountdownTimer';
+
+export default function ApplicationControls() {
+  const [eventDate, setEventDate] = useState('');
+  const [appId, setappId] = useState('');
+  const [startDate, setStartDate] = useState('');
+  const [endDate, setEndDate] = useState('');
+  const [alert, setAlert] = useState({ type: '', message: '', isVisible: false });
+
+  const { mutate: control } = useMutate({
+    path: `/api/admin/app-controls/${appId}`,
+    verb: 'PATCH',
+  });
+
+  const { data: eventData } = useGet({
+    path: '/api/admin/app-controls',
+    verb: 'GET',
+  });
+
+  useEffect(() => {
+    if (eventData) {
+      setEventDate(eventData.startAt);
+      setappId(eventData.id);
```

Comment:
```suggestion
      setEventDate(eventData.startAt);
      setappId(eventData.id);
      setStartDate(eventData.startAt);
      setEndDate(eventData.endAt);
```

Reply:
setting startDate and endDate here will make them show up like a default in the date picker

Reply:
also not sure we need eventDate if we do my other suggestion